README.md

```diff
@@ -45,7 +45,8 @@ Please find those that resonate with you, donate time, money, or both.
 - [Mexican American Legal Defense and Education Fund](http://www.maldef.org/)
 - [Muslim Justice League](https://www.muslimjusticeleague.org)
 - [Workers Defense Project](http://www.workersdefense.org/)
-- [Asian Family Support Services of Austin](http://www.saheli-austin.org/)
+- [Lawyers' Committee for Civil Rights Under Law](https://lawyerscommittee.org/)
+- [Southern Christian Leadership Conference](http://nationalsclc.org/)
```

Comment:
the SCLC organizes seminars/workshops in communities where human trafficking occurs to help them address root causes, but I think historically they've been associated with African American civil rights causes.

They're actual impact seems a little abstract to me ("raising awareness" and such), which is certainly less appealing than the other organizations you identified that provide concrete services for people. Also, you're right, the religious overtone makes it exclusive. Removing